neptune/alpha/internal/backends/hosted_neptune_backend.py

```diff
@@ -228,8 +228,8 @@ def _execute_operations(self,
         try:
             result = self.leaderboard_client.api.executeOperations(**kwargs).response().result
             return [MetadataInconsistency(err.errorDescription) for err in result]
-        except HTTPNotFound:
-            raise ExperimentUUIDNotFound(exp_uuid=experiment_uuid)
+        except HTTPNotFound as e:
```

Comment:
Do we want to update all exception raising code in `except` blocks now?

Reply:
Yes

Reply:
It would be good to have it but I do not see a reason to do it everywhere in this merge request. Let's keep it clean.